presto-main/src/main/java/com/facebook/presto/sql/planner/Plan.java

```diff
@@ -21,14 +22,17 @@
 {
     private final PlanNode root;
     private final TypeProvider types;
+    private final StatsAndCosts statsAndCosts;
```

Comment:
I'm going to leave it in, since they are tied, and we use Plan.getStatsAndCost in a few places (not just to carry it through to fragments but also to call plan printer on the Plan itself).